pkg/management/component_test.go

```diff
@@ -1291,9 +1291,11 @@ func TestExecuteActionsEmail(t *testing.T) {
 
 	// Send email actions
 	{
+
 		mockKeycloakClient.EXPECT().ExecuteActionsEmail(accessToken, realmName, userID, actions, key1, value1, key2, value2).Return(nil).Times(1)
 
 		var ctx = context.WithValue(context.Background(), cs.CtContextAccessToken, accessToken)
+		mockEventDBModule.EXPECT().ReportEvent(ctx, "INIT_PASSWORD", "back-office", gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
```

Comment:
We could have 2 calls to ExecuteActionsEmail in this test : it would check that ReportEvent is called only if sms-password-set is sent to ExecuteActionsEmail